### DISCUSSION TITLE
Enable me to showcase my DEV blog on my profile

### DISCUSSION THREAD

**Discussion by andypiper** [dailydotdev/daily#637] [Feature Requests]
Right now there are Social options for Twitter, GitHub, Hashnode, and Website. 

I write primarily on DEV.to (rather than Hashnode, where I have an account, but have never posted), and I'd like to be able to link to my page there from my dailydotdev profile. 

Relatedly, I don't see a lot of content from DEV in the feeds, which is disappointing - I've posted a few things from others as submissions myself, but there's some great content that gets missed on dailydotdev.

**Comment by idoshamun** [comment]
Hi @andypiper,

First of all, thanks for the suggestion and being such an avid user on daily.dev. I love to see your picks! 😁 
As for your suggestion, we will add the DEV.to profile as well. It might take some time as we're revamping our authentication system at the moment.
We crawl DEV's articles using their API but add to our system only articles that passed a certain number of engagement. DEV has a lot of content that gets published daily, and we can't add everything, unfortunately. I will revisit our threshold though
